GVFS/GVFS.Service/Handlers/NotificationHandler.Mac.cs

```diff
@@ -0,0 +1,27 @@
+﻿using GVFS.Common.NamedPipes;
+using GVFS.Common.Tracing;
+using System;
+
+namespace GVFS.Service.Handlers
+{
+    public class NotificationHandler
+    {
+        // NotificationHandler uses a singleton so in the future, we can create callback actions
+        // from responses sent by GVFS.Service.UI when a user clicks on a notification.
+        private static NotificationHandler instance = new NotificationHandler();
+
+        private NotificationHandler()
+        {
+        }
+
+        public static NotificationHandler Instance
+        {
+            get { return instance; }
+        }
+
+        public void SendNotification(ITracer tracer, int sessionId, NamedPipeMessages.Notification.Request request)
+        {
+            throw new NotImplementedException();

```

Comment:
Will this exception get thrown on Mac every time we hit this line of code:

```
this.SendNotification(sessionId, "GVFS AutoMount", "Attempting to mount {0} GVFS repo(s)", activeRepos.Count);
```

In `RepoRegistry.AutoMountRepos`?

Or does this PR not actually enable auto-mount on Mac?

Reply:
This PR does not enable auto-mount